### PR TITLE
CNSM deploy csv update

### PR DESCRIPTION
**Pull request by rtravis-whoi**
Added note for SWND2 on CNSM-15 deployment - sensor raised to same height as FDCHP.  Can confirm via CP01CNSM-00015.xlsx config sheet (METBK tab - height above waterline) in Vault.

### CODE REVIEW

**Comment by rtravis-whoi** [comment]
[Surface Mooring Wind Sensor Heights [DT-214]](https://trello.com/c/KavGDFE6/214-surface-mooring-wind-sensor-heights-dt-214)

File: deployment/CP01CNSM_Deploy.csv [hunk 1]
```diff
@@ -717,7 +717,7 @@ AR61,,AR66,,CP01CNSM-SBD12-06-METBKA003,15,1,2021-11-03T14:45:00,2022-04-13T12:0
 AR61,,AR66,,CP01CNSM-SBD12-06-METBKA004,15,1,2021-11-03T14:45:00,2022-04-13T12:05:00,CGMCP-01CNSM-00015,,CGINS-METLWR-00267,40.139823,-70.771173,,-4,132,,,
 AR61,,AR66,,CP01CNSM-SBD12-06-METBKA005,15,1,2021-11-03T14:45:00,2022-04-13T12:05:00,CGMCP-01CNSM-00015,,CGINS-METPRC-00243,40.139823,-70.771173,,-4,132,,,
 AR61,,AR66,,CP01CNSM-SBD12-06-METBKA006,15,1,2021-11-03T14:45:00,2022-04-13T12:05:00,CGMCP-01CNSM-00015,,CGINS-METSWR-00247,40.139823,-70.771173,,-4,132,,,
-AR61,,AR66,,CP01CNSM-SBD12-06-METBKA007,15,1,2021-11-03T14:45:00,2022-04-13T12:05:00,CGMCP-01CNSM-00015,,CGINS-METWND-00268,40.139823,-70.771173,,-5,132,,,
+AR61,,AR66,,CP01CNSM-SBD12-06-METBKA007,15,1,2021-11-03T14:45:00,2022-04-13T12:05:00,CGMCP-01CNSM-00015,,CGINS-METWND-00268,40.139823,-70.771173,,-5,132,sensor raised to same height as FDCHP,,
```
Comment (cooleyky) [inline deployment/CP01CNSM_Deploy.csv:720]: Looks good

**Review by cooleyky — APPROVED** [review]
Looks good!

**Review by sauuyer — APPROVED** [review]
comment looks good

**Pull request merged**